guides/v3.7.0/reference/syntax-conversion-guide.md

```diff
@@ -52,23 +55,57 @@ You can apply regular HTML attributes like `class`, `id`, `aria-role`, etc. when
+### Positional parameters
 
-The fundamental change is that the scope of arguments passed in and properties local to the component are no longer mashed together. There is a clear boundary between arguments passed down from the parent and arguments that is tracked in the local component.
+The classic invocation syntax supports passing arguments to the component by their position.
+In the following example, `"first"` and `"second"` are positional parameters:
 
 ```handlebars
-{{@name}} {{!-- this is the arg passed down from the parent --}}
-{{this.name}} {{!-- this is the property that is tracked in the local component js --}}
+{{my-greeting "Hello" "World"}}
 ```
 
-Variables passed into a component have an `@` before them, also known as [named arguments](https://github.com/emberjs/rfcs/blob/master/text/0276-named-args.md). While variables created by the current component will have `this` in front of it.
+As shown in the relevant ["Position Params"](https://guides.emberjs.com/release/components/passing-properties-to-a-component/#toc_positional-params) part of the Guides,
+there are two ways to handle them inside the component.
+One way is to individually specify what component property the positional parameter should map to.
+The other way is to map all positional parameters to the `params` property and refer to them by their index.
 
-### Limitations
+#### Individual names
 
-- Positional arguments like `{{foo-bar "first" "second"}}` cannot be used with Angle Brackets.
-- Angle Bracket syntax requires using the `@myvariablename` notation when passing a variable into a component, with the variable name being lowercase
-- User defined components must be capitalized such as `<FooBar></FooBar>` or `<FooBar />`
+If `my-greeting` had the following implementation:
 
-### What is happening with the Classical Invocation Syntax?
+```javascript {data-filename="app/components/my-greeting.js"}
+import Component from '@ember/component';
 
-The Classical Invocation Syntax – which uses “curlies” `{{}}` instead of angle brackets `<>` – is here to stay, the ability to accept positional arguments and "else" blocks makes them ideal for control-flow like components such as `{{liquid-if}}`.
+export default Component.extend({
+}).reopenClass({
+  positionalParams: ['greeting', 'name']
+});
+```
+
+To invoke it using the angle bracket syntax, you would do the following:
+
+```handlebars
+<MyGreeting @greeting="Hello" @name="World" />
+```
+
+#### `params` array
+
+If `my-greeting` has the following implementation:
+
+```javascript {data-filename="app/components/my-greeting.js"}
+import Component from '@ember/component';
+
+export default Component.extend({
+}).reopenClass({
+  positionalParams: 'params'
+});
+```
+
+```handlebars
+<MyGreeting @params={{array "Hello" "World"}}>
+```
+
+### What is happening with the Classic Invocation Syntax?
+
+Classic invocation syntax—which uses “curlies” `{{}}` instead of angle brackets `<>`–is here to stay.
```

Comment:
is there a reason you have "curlies" here instead of handlebars? handlebars and angle brackets seem more consistent with each other.

Reply:
I kept it from the original, will update.